libs/jump_portrait/src/jump_portrait/fetch.py

```diff
@@ -84,7 +85,7 @@ def get_jump_image(
         Site identifier (also called foci), default is 1.
     correction : str
         Whether or not to use corrected data. It does not by default.
-    apply_illum : bool
+    apply_correction : bool
         When correction=="Illum" apply Illum correction on original image.
```

Comment:
(I forgot to update the argument description). Please make it "When apply_correction==...."